content/deploy/community-cloud/manage-your-app/edit-your-app.md

```diff
@@ -15,72 +15,69 @@ Spin up a cloud-based development environment for your deployed app in minutes.
 
 ### Create a codespace for your deployed app
 
-1. From your workspace at <a href="https://share.streamlit.io" target="_blank">share.streamlit.io</a>, click the overflow icon (<i style={{ verticalAlign: "-.25em" }} className={{ class: "material-icons-sharp" }}>more_vert</i>) next to your app. Click "**Edit**."
+1. From your workspace at <a href="https://share.streamlit.io" target="_blank">share.streamlit.io</a>, click the overflow icon (<i style={{ verticalAlign: "-.25em" }} className={{ class: "material-icons-sharp" }}>more_vert</i>) next to your app. Click "**Edit with Codespaces**."
 
    ![Edit your app with GitHub Codespaces](/images/streamlit-community-cloud/workspace-app-edit.png)
 
-2. A `.devcontainer/devcontainer.json` file will be added to your repository. If you already have a file of the same name in your repository, it will not be changed. You may delete or rename your existing devcontainer configuration if you would like your repository to receive the instance created by Streamlit Community Cloud.
+   Community Cloud will add a `.devcontainer/devcontainer.json` file to your repository. If you already have a file of the same name in your repository, it will not be changed. You may delete or rename your existing devcontainer configuration if you would like your repository to receive the instance created by Community Cloud.
```

Comment:
```suggestion
   Community Cloud will add a `.devcontainer/devcontainer.json` file to your repository. If you already have a file of the same name in your repository, it will not be changed.  If you want your repository to receive the instance created by Community Cloud, delete or rename your existing devcontainer configuration.
```